docs/source/guide/get_started/quick_start_guide/cli_commands.rst

```diff
@@ -150,7 +150,7 @@ Training
 - ``weights.pth`` - a model snapshot
 - ``label_schema.json`` - a label schema used in training, created from a dataset
 
-The results will be saved in ``./model`` folder by default. The output folder can be modified by ``--save-model-to`` option. These files are used by other commands: ``export``, ``eval``, ``demo``, etc.
+The results will be saved in ``./model`` folder by default. The output folder can be modified by ``--output`` option. These files are used by other commands: ``export``, ``eval``, ``demo``, etc.
```

Comment:
```suggestion
The results will be saved in ``./outputs`` folder by default. The output folder can be modified by ``--output`` option. These files are used by other commands: ``export``, ``eval``, ``demo``, etc.
```

Reply:
In my understanding, the final output is made in `outputs`. and there are `2023000_00000/logs`, `2023000_00000/models` in the `outputs` folder. So, above statement needs to be changed

Reply:
Yes, I agree. thanks! I'll fix it, and also try to add unit-test (maybe should add it to integration/cli test). I think `-o` would be nice to have for convenience, what do you think? I can fix the documentation, or I'll delete it as you suggest.

Reply:
Editing documentation is also an excellent way.

Reply:
Yeah, I'm only going to modify the --help output part of the cli_commands part, I think the example can be left as is.
Thanks for comment :)

Reply:
i added integration test cases related this PR https://github.com/openvinotoolkit/training_extensions/pull/1946/commits/673f268f937df627cb15aa06f4153461c965d882